### ISSUE TITLE
Firestore serverTimestamp() not working

### ISSUE THREAD

**Issue by mirkonasato** [invertase/react-native-firebase#474]
Trying to set a document field to be the server timestamp with Firestore, following the [example in the Firebase documentation](https://firebase.google.com/docs/firestore/manage-data/update-data#update_fields_in_nested_objects):

```javascript
import firebase from 'react-native-firebase';
//...
messagesRef.add({
  time: firebase.firestore.FieldValue.serverTimestamp(),
  //...
};
```

Looking at the new document in the Firebase console what gets inserted is not actually of type `timestamp`.

When running the code on Android the field is of type `string` with value `com.google.firebase.firestore.FieldValue$zzb@4c6b528`.

When adding the document from iOS it's of type `object` and apparently empty. (And trying to read that document from Android causes the app to crash with `RuntimeException: Cannot convert object of type {}`.)

### Environment

1. Application Target Platform:
both Android and iOS

2. Development Operating System: 
macOS High Sierra

3. Build Tools: 

4. React Native version: 
0.49.1

5. RNFirebase Version: 
3.0.0

6. Firebase Module: 
Firestore

**Comment by chrisbianca** [comment]
@mirkonasato Thanks for reporting, it's something we're aware of.  We'll look to have this supported in a release next week.

**Comment by chrisbianca** [comment]
I have just released this as part of v3.0.3
